docs/pages/guides/keyboard-handling.mdx

```diff
@@ -0,0 +1,291 @@
+        renderItem={({ item }) => <MessageItem message={item} />}
+        keyExtractor={item => item.createdAt.toString()}
+        contentContainerStyle={styles.listStyle}
+      />
+      <TextInput placeholder="Type a message..." style={styles.textInput} />
+      <Animated.View style={fakeView} />
+    </View>
+  );
+}
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    paddingTop: Platform.OS === "android" ? StatusBar.currentHeight : 0,
+  },
+  listStyle: {
+    padding: 16,
+    gap: 16,
+  },
+  textInput: {
+    width: "95%",
+    height: 45,
+    borderWidth: 1,
+    borderRadius: 8,
+    borderColor: "#d8d8d8",
+    backgroundColor: "#fff",
+    padding: 8,
+    alignSelf: "center",
+    marginBottom: 8,
+  },
+});
+```
+
+### Handling multiple inputs
+
+For screens with multiple inputs, the Keyboard Controller provides `KeyboardAwareScrollView` and `KeyboardToolbar` components. These components handle input navigation and prevent the keyboard from covering the screen without custom configuration:
+
+```tsx FormScreen.tsx
+import { TextInput, View, StyleSheet } from 'react-native';
+import { KeyboardAwareScrollView, KeyboardToolbar } from 'react-native-keyboard-controller';
+
+export default function FormScreen() {
+  return (
+    <>
+      <KeyboardAwareScrollView bottomOffset={62} contentContainerStyle={styles.container}>
+        <View>
+          <TextInput placeholder="Type a message..." style={styles.textInput} />
+          <TextInput placeholder="Type a message..." style={styles.textInput} />
+        </View>
+        <TextInput placeholder="Type a message..." style={styles.textInput} />
+        <View>
+          <TextInput placeholder="Type a message..." style={styles.textInput} />
+          <TextInput placeholder="Type a message..." style={styles.textInput} />
+          <TextInput placeholder="Type a message..." style={styles.textInput} />
+        </View>
+        <TextInput placeholder="Type a message..." style={styles.textInput} />
+      </KeyboardAwareScrollView>
+      <KeyboardToolbar />
+    </>
+  );
+}
+
+const styles = StyleSheet.create({
+  container: {
+    gap: 16,
+    padding: 16,
+  },
+  listStyle: {
+    padding: 16,
+    gap: 16,
+  },
+  textInput: {
+    width: 'auto',
+    flexGrow: 1,
+    flexShrink: 1,
+    height: 45,
+    borderWidth: 1,
+    borderRadius: 8,
+    borderColor: '#d8d8d8',
+    backgroundColor: '#fff',
+    padding: 8,
+    marginBottom: 8,
+  },
+});
+```
+
+The above example wraps the inputs with `KeyboardAwareScrollView` to prevent the keyboard from covering them. The `KeyboardToolbar` component displays navigation controls and a dismiss button. While it works without configuration, you can customize the toolbar content if needed.
+
+## Additional resources
+
+- [Keyboard guide source code](https://github.com/betomoedano/keyboard-guide)
+- [Keyboard Controller documentation](https://kirillzyusko.github.io/react-native-keyboard-controller)
```

Comment:
Presentation nit, we can convert those inline links to `BoxLink`s.

```suggestion
- [Sample project based on this guide](https://github.com/betomoedano/keyboard-guide)
- [`react-native-keyboard-controller` documentation](https://kirillzyusko.github.io/react-native-keyboard-controller)
```

Also, small suggestion to the wording, and in case of docs link, it seems more clear to use the package name, since  "Keyboard Controller" sounds quite generic.